pkg/features/kube_features.go

```diff
@@ -314,8 +314,9 @@ const (
 	// Make the kubelet use shutdown configuration based on pod priority values for graceful shutdown.
 	GracefulNodeShutdownBasedOnPodPriority featuregate.Feature = "GracefulNodeShutdownBasedOnPodPriority"
 
-	// owner: @arjunrn @mwielgus @josephburnett
+	// owner: @arjunrn @mwielgus @josephburnett @sanposhiho
```

Comment:
Can we add the link to the KEP in whilst we're here, as per: https://github.com/kubernetes/kubernetes/blob/067b8a80255de4e1d6d51c6eae525c1e673eb01c/pkg/features/kube_features.go#L30

Reply:
👍 
Fixed.